src/store/modules/config.js

```diff
@@ -38,9 +38,9 @@ export const config = {
   mutations: {
     setFullConfig(state, payload) {
       // Clear existing state properties
-      Object.keys(state).forEach((key) => {
-        Vue.delete(state, key);
-      });
+      // Object.keys(state).forEach((key) => {
+      //   Vue.delete(state, key);
+      // });
```

Comment:
why is this being removed? it seems like it would break functionality to not remove old keys before trying to reset the config

Reply:
@dacoaster , getConfig function gave me this result.
![Config_error_1](https://github.com/yatt-ai/yattie/assets/146516559/48a8faaf-4a1a-413e-835f-806757171817)

I think the config need to be like this.
![Config_error_2](https://github.com/yatt-ai/yattie/assets/146516559/86c18ad7-ba7b-4ada-bbd5-cef167b55a10)

Reply:
Following up on this.

Reply:
I have reverted the code to original.
There was an error getting config data from Backend.
It has solved.
I removed the comments.